draft-ietf-scitt-architecture.md

```diff
@@ -74,12 +79,9 @@ normative:
   RFC9360:
   IANA.params:
   IANA.cose:
-  COSWID:
-    target: https://www.rfc-editor.org/rfc/rfc9393
-    title:  COSWID Specification
-  CWT_CLAIM_COSE:
-    target: https://datatracker.ietf.org/doc/draft-ietf-cose-cwt-claims-in-headers/
-    title: CBOR Web Token (CWT) Claims in COSE Headers
+  RFC9393: COSWID
+  I-D.ietf-cose-cwt-claims-in-headers: CWT-CLAIMS-COSE
```

Comment:
Per the [`kramdown-rfc` docs]() and local testing, you need to flip the key and value.

```suggestion
  COSWID: RFC9393
  CWT_CLAIMS_COSE: I-D.ietf-cose-cwt-claims-in-headers
```

Tested this locally in the same container as GHA by using this branch, running `make clean`, then `make. This suggestion will remove [the relevant build failures](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/actions/runs/7125650198/job/19401978042#step:6:35) and allow it to publish once reviewed and merged by others.